### PR TITLE
Create pipeline.travis.yml

### PR DESCRIPTION
**Pull request by dthadi3**
Created pipeline.travis.yml and added support for power and intel.

### CODE REVIEW

**Review by gururajrkatti — COMMENTED** [review]
Are there no tests in this ? Did you try "make check" or "make tests" ?

**Comment by dthadi3** [comment]
yeah tried to run, didn't have any test cases so removed make test.
$ make test
make: Nothing to be done for 'test'.

**Comment by gururajrkatti** [comment]
Can you pl. check any relevant test related target from Makefile

**Comment by dthadi3** [comment]
@gururajrkatti please review the latest code, if it looks fine I'll squash the commits.

File: travis-ymls/pipeline.travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 #
 # Package             : pipeline
 # Source Repo         : https://github.com/tektoncd/pipeline.git
-# Travis Job Link     : https://travis-ci.com/github/dthadi3/pipeline/builds/217320099
+# Travis Job Link     : https://travis-ci.com/github/dthadi3/pipeline/builds/217856077
```
Comment (gururajrkatti) [inline travis-ymls/pipeline.travis.yml:5]: I see some failures in the test logs such as "TestFetch".

Comment (dthadi3) [reply travis-ymls/pipeline.travis.yml:5]: As per my assumption it is checking for git which got failed status and remaining everything is same as intel log only.
Any additional inputs on this @gururajrkatti  @seth-priya

**Comment by dthadi3** [comment]
@gururajrkatti anything else required on this?

**Review by gururajrkatti — APPROVED** [review]
ok

**Pull request merged**